docs/config/README.md

```diff
@@ -1457,25 +1457,33 @@ format = '[+$added]($added_style)/[-$deleted]($deleted_style) '
 The `git_status` module shows symbols representing the state of the repo in your
 current directory.
 
+::: tip
+
+The Git Status module is very slow in Windows directories (for example under `/mnt/c/`) when in a WSL environment.
+You can disable the module or use the `windows_starship` option to use a Windows-native Starship executable to compute `git_status` for those paths.
```

Comment:
Do we want to add an example here or would that clutter things too much? (e.g. for a system-wide install, `/mnt/c/Program Files/starship/starship.exe` or something of that nature)

Reply:
I think the tips already look fairly cluttered, and I already added an example below.